src/FNA3D_Driver_Vulkan.c

```diff
@@ -11498,6 +11505,11 @@ static uint8_t VULKAN_SupportsNoOverwrite(FNA3D_Renderer *driverData)
 	return 1;
 }
 
+static uint8_t VULKAN_SupportsSRGBFrameBuffer(FNA3D_Renderer *driverData)
+{
+	return 1;
```

Comment:
We should probably check this with vkGetPhysicalDeviceFormatProperties (see S3TC's query, but with different bitflags).